README.md

```diff
@@ -47,6 +47,8 @@ user_id: <Rigetti Forest User ID>
 
 If `url` is not specified, it will default to `https://api.rigetti.com/qvm`.
 
+You can also use the `config_setup.py` script to assist in creating the configuration file.
```

Comment:
Can you be a bit more verbose on this? How do I use the script, what information do I need etc.